### DISCUSSION TITLE
加强迪卢克老爷

### DISCUSSION THREAD

**Discussion by theweakest114** [share121/inter-knot#2079] [其他]
![微信截圖_20240814151249](https://github.com/user-attachments/assets/d58d0aea-24aa-46d2-923d-57f804129da9)
4年了，原神连刻晴都加强了，凭什么我们卢姥爷不能加强

**Comment by break-under-star** [comment]
带芙宁娜和闲云，开大直接开跳，一下10W<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by gitmolin** [comment]
同意楼上，加个六命班，开大都省了
